fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/OCFLPersistentStorageSession.java

```diff
@@ -129,6 +125,14 @@ protected OCFLPersistentStorageSession(final String sessionId, final FedoraToOCF
         this.fedoraOcflIndex = fedoraOcflIndex;
         this.objectSessionFactory = objectSessionFactory;
         this.sessionMap = new ConcurrentHashMap<>();
+
+        //load the persister list if empty
+        persisterList.add(new CreateRDFSourcePersister(this.fedoraOcflIndex));
+        persisterList.add(new UpdateRDFSourcePersister(this.fedoraOcflIndex));
+        persisterList.add(new CreateNonRdfSourcePersister(this.fedoraOcflIndex));
+        persisterList.add(new UpdateNonRdfSourcePersister(this.fedoraOcflIndex));
+        persisterList.add(new DeleteResourcePersister(this.fedoraOcflIndex));
```

Comment:
I'm not sure if this makes difference, just wondering why we moved away from a static list of persisters to an instance list?